app/views/user_mailer/notify_subscriber.html.haml

```diff
@@ -1,5 +1,6 @@
-Hi, #{@user.name},
+From: #{@sender}
```

Comment:
I would say to send `current_user` and change this to `@sender.name` so it's easier to understand.